lib/secretariat/invoice.rb

```diff
@@ -106,7 +106,7 @@ def namespaces(version: 1)
       )
     end
 
-    def to_xml(version: 1, skip_validation: false, mode: :zugferd)
+    def to_xml(version: 1, skip_validation: false, mode: :zugferd, payment: {})
```

Comment:
I think the payment info should be part of the invoice struct and not be passed as an option here. I think if there is something like `:payment_terms` with the elements as used below in the struct it fits better.